packages/cli/src/config/rollup.config.js

```diff
@@ -37,9 +37,28 @@ function greenwoodHtmlPlugin(compilation) {
 
   return {
     name: 'greenwood-html-plugin',
-    load(id) {
-      if (path.extname(id) === '.html') {
-        return '';
+    async load(id) {
+      const extension = path.extname(id);
+      
+      if (extension === '.html') {
+        return Promise.resolve('');
+      }
+
+      // handle custom user file extensions
+      const customResources = compilation.config.plugins.filter((plugin) => {
```

Comment:
Nice! Was curious how we were going to implement this in rollup.  This is exactly whats needed.